README.md

```diff
@@ -50,23 +50,14 @@ For the latest information on setting up a development environment, see the [Dev
 
 ### Mac M1/ARM64 [EXPERIMENTAL]
```

Comment:
I'd say we are good to remove the "experimental" tag from this. We have multiple developers running it compiled natively for ARM for months without issue.